### ISSUE TITLE
bug: Cursor keys not working properly in "pcms shell"

### ISSUE THREAD

**Issue by robvdl** [robvdl/pyramidcms#11]
Using the cursor keys is printing escape codes for me (^[[D, ^[[C, etc..)

However when opening a standard python shell I don't get this.

**Comment by robvdl** [comment]
I checked it out, doesn't fix it for me

**Comment by johndanielcurley** [comment]
Tested working on Fedora 21, Python 3.4.1. Can you please give error messages as I can't replicate bug. Have updated script according to this:

http://stackoverflow.com/questions/19754458/open-interactive-python-console-from-a-script

**Comment by johndanielcurley** [comment]
Also did you merge or test the branch?

**Comment by johndanielcurley** [comment]
I'll set up an ubuntu 14.04 vm to test if it's not fixed. Seems really odd.

**Comment by robvdl** [comment]
It's merged, it's fixed now and working fine for me.

Also I removed Windows support, I don't care for ever serving a CMS app on a Windows server, ever...

I removed the new import it introduced (pyreadline) as it was only needed for Windows, also it would have needed to be in requirements/base.txt not dev.txt because we also need to be able open a shell on production.  Anyway, pyreadline is now gone as it was only for Windows.
